domain/transfer.go

```diff
@@ -171,3 +174,36 @@ func (TransferProcessHandler) HandleEvent(
 
 	return nil
 }
+
+// HandleTimeout handles a timeout message that has been routed to this handler.
+func (TransferProcessHandler) HandleTimeout(
+	ctx context.Context,
+	s dogma.ProcessTimeoutScope,
+	m dogma.Message,
+) error {
+	switch x := m.(type) {
+	case TransferReadyToSend:
+		s.ExecuteCommand(commands.DebitAccount{
+			TransactionID:   x.TransactionID,
+			AccountID:       x.FromAccountID,
+			TransactionType: messages.Transfer,
+			Amount:          x.Amount,
+			ScheduledDate:   x.ScheduledDate,
+		})
+
+	default:
+		panic(dogma.UnexpectedMessage)
+	}
+
+	return nil
+}
+
+// TransferReadyToSend is a timeout message notifiying that the transfer is
+// ready to send.
+type TransferReadyToSend struct {
```

Comment:
Is "send" in our existing ubiquitous language?  (I'm genuinely not sure, but I don't think so)

Reply:
We didn't really have a word for this in the ubuiqitious language. The closesed we have is your earlier comment. But now that you mention it, the next step that it performs is *Debit*. So perhaps this could be `TransferReadyToDebit`. But is that really what the user would have scheduled, the debit side of it?

Reply:
I had the same thought. Though, I think it probably *is* better than "send".

Reply:
How about just `TransferReadyToProceed`.